data/korath/korath hails.txt

```diff
@@ -0,0 +1,1470 @@
+	phrase
+		"exile humanika: You will give us"
+	word
+		" "
+	phrase
+		"exile humanika: non-luxury foodstuffs" 3
+		"exile humanika: raw materials" 2
+		"exile humanika: tactical and strategic items"
+	word
+		" "
+	phrase
+		"exile humanika: and we will let you live."
+
+phrase "exile humanika: You will give us"
+	word
+		"Can haf"                # Can have (mispronounced)
+		"Will haf"               # Will have (mispronounced)
+		"Gif us"                 # Give us (mispronounced)
+		"Tribut'"                # Tribute (mispronounced, probably heard it from pirates)
+		"Wee taek yur"           # We take your (mispronounced)
+		"Is are"                 # Is our (mispronounced)
+
+phrase "exile humanika: All of your"
+	word
+		"All yer"                # All your (mispronounced)
+		"Yor"                    # Your (mispronounced)
+		"Te"                     # The (mispronounced)
+		"A' of our"              # All of your (mispronounced)
+
+phrase "exile humanika: is ours now."
+	word
+		"blong to 's."           # belong to us (mispronounced)
+		"giv mine."              # give mine (mispronounced)
+		"arrr ows now."          # are ours now (mispronounced)
+		"ar b'long to us."       # are belong to us (mispronounced)
+
+phrase "exile: You will give us"
+	word
+		"Asra rata ilehfef ifat"     # You will give us
+		"Ifat kokroref asra"         # We demand your
+		"Itimif aofe' oftimif"       # [We now] own all of your
+		"Kohelef ranraneb poreb"     # [We are] seizing a full [cargo] hold of
+		"Ratileh ifat aofe'"         # Give us all [of your]
+		"Ratomlebis tokro mihfanref" # [It is] necessary to take [your]
+		"Pakrileb rapakleh bosip nrag akit mihfa" # [Our] survival dictates we take
+
+phrase "exile mixed: You will give us"
+	phrase
+		"exile: You will give us" 2
+		"exile humanika: You will give us"
+
+phrase "exile mixed: food"
+	phrase
+		"exile: good food" 2
+		"exile humanika: non-luxury foodstuffs"
+
+# Wish list food, probably half joking
+phrase "exile: good food"
+	word
+		"asame' oavip aok lane'"       # fresh dark green lettuce
+		"nrahfot tasah pagrel"         # crunchy rhino beetles
+		"ilehulk' lane' nrag nrakaf"   # ripe succulent strawberries
+		"sud nrannrab ranres nracah pikob" # baked cod with garlic
+		"kemmij emu' nrannrab"         # baked sole fillet
+		"'ah koam imipd"               # Chilean Sea Bass
+		"rati is 'aka 'aklet"          # fish en papillote
+		"rumef rohfe osaf"             # honeydew melons
+		"meba ak'eb nrag pagrel"       # crisp gala apples
+		"oham nrirot nradrid 'omap"    # finely chopped yellow squash
+		"ica' oavip kolab"             # shredded collard greens
+		"kamo bosip nrag kaimennref"   # live squirming mealworms
+		"totnrafe' aqlam nridad nramen" # a bowlful of hornworm larvae
+		"polome' irumut kofe'"         # an entire ant colony
+		"ratobfok la'eh nridad rimim"  # a big heaping pile of waxworms
+		"nrala' lomek nroloh akro"     # a nest of ostrich eggs
```

Comment:
Deleting four items from each list to show some variation between the two cultures and compensate for the six I added.

```suggestion
		"oham nrirot nradrid 'omap"    # finely chopped yellow squash
		"ica' oavip kolab"             # shredded collard greens
		"totnrafe' aqlam nridad nramen" # a bowlful of hornworm larvae
		"polome' irumut kofe'"         # an entire ant colony
```